assets/scripts/lobby/options.js

```diff
@@ -275,6 +275,33 @@ var userOptions = {
         }
     },
 
+   optionProposedTeamIcon: {
+        defaultValue: "false",
+        onLoad: function(){
+            if (docCookies.getItem("optionProposedTeamIcon") === "shield") {
+                $("#option_proposed_team_icon")[0].checked = true;
+                updateGunImage("shield");
+            }
+            else{
+                $("#option_proposed_team_icon")[0].checked = false;
+                updateGunImage("gun");
+            }
+        },
+        initialiseEventListener: function(){
+            $("#option_proposed_team_icon")[0].addEventListener("click", function () {
+                //when they press it...
+                var checked = $("#option_proposed_team_icon")[0].checked;
+                if(checked === true){
+                    updateGunImage("shield");
+                } else {
+                    updateGunImage("gun");
+                }
+                //save their option in cookie
```

Comment:
You're not saving their new option in the cookie

Reply:
Forgot to delete that comment line. I did put the saving task inside updateGunImage()